service/github/github.go

```diff
@@ -309,13 +309,21 @@ func buildSingleSuggestion(c *reviewdog.Comment, s *rdf.Suggestion) (string, err
 	if start.GetColumn() > 0 || end.GetColumn() > 0 {
 		return buildNonLineBasedSuggestion(c, s)
 	}
+
+	// A test suggestion that uses four backticks w/o code fence block.
+	// Fixes: https://github.com/reviewdog/reviewdog/issues/999
+	txt := s.GetText()
+	backticks := commentutil.GetCodeFenceLength(txt)
+
 	var sb strings.Builder
-	sb.WriteString("```suggestion\n")
-	if txt := s.GetText(); txt != "" {
+	sb.Grow(backticks + len("suggestion\n") + len(txt) + len("\n") + backticks)
+	commentutil.WriteCodeFence(&sb, backticks)
```

Comment:
Can you check the error?

Reply:
I think this warning is false positive.
We don't need to check return values because the methods of `strings.Builder` always return nil error.

https://pkg.go.dev/strings#Builder.Write

> Write appends the contents of p to b's buffer. Write always returns len(p), nil.

https://pkg.go.dev/strings#Builder.WriteByte

> WriteByte appends the byte c to b's buffer. The returned error is always nil.

The return values exist only to satisfy well-known interfaces such as `io.Writer`, `io.ByteWriter`, etc.

Should I suppress the warning?

```suggestion
	_ = commentutil.WriteCodeFence(&sb, backticks)
```

Reply:
ok, sounds good. You can leave it as is.

Reply:
Thanks!